app/javascript/src/components/Dashboard/Notes/List.jsx

```diff
@@ -0,0 +1,55 @@
+import React, {useState} from 'react'
+
+import Note from './Note';
+import EditNotePane from './Pane/Edit';
+import DeleteAlert from "./DeleteAlert";
+
+const List = ({notes, fetchNotes}) => {
+
+  const [showEdit, setShowEdit] = useState(false);
+  const [selectedNote, setSelectedNote] = useState({});
+
+  const [showDeleteAlert, setShowDeleteAlert] = useState(false);
+  const [selectedNoteId, setSelectedNoteId] = useState([]);
+
+  const onClickEditHandler = (selectedNote) => {
```

Comment:
Are you bypassing git hooks? Because I believe our Eslint+Prettier hooks would have corrected the mistake with this line.